fdbserver/Status.actor.cpp

```diff
@@ -3670,6 +3671,37 @@ ACTOR Future<StatusReply> clusterGetStatus(
 		    .detail("StatusSize", statusObj.getFinalLength());
 
 		return StatusReply(statusObj.getJson());
+
+	} catch (Error& e) {
+		TraceEvent(SevError, "StatusError").error(e);
+		throw;
+	}
+}
+
+StatusReply clusterGetFaultToleranceStatus(const std::string& statusStr) {
+	double tStart = timer();
+
+	try {
+		json_spirit::mValue mv = readJSONStrictly(statusStr);
+		JSONDoc jsonDoc(mv);
+
+		std::string faultToleranceRelatedFields[] = {
+			"fault_tolerance", "data",    "logs", "maintenance_zone", "maintenance_seconds_remaining", "qos",
```

Comment:
is this formatted correctly? I see extra spaces between "data" and "logs", and the line below.

Reply:
That's "clang-format" adding those extra spaces.